codesign/bin/codesign.dart

```diff
@@ -81,16 +81,14 @@ Future<void> main(List<String> args) async {
           'the name of the certificate for flutter, for example, is: FLUTTER.IO LLC',
     )
     ..addOption(
-      kGcsDownloadPathOption,
-      help: 'The google cloud bucket path to download the artifact from\n'
-          'e.g. supply `--gcs-download-path=gs://flutter_infra_release/ios-usb-dependencies/unsigned/ios-deploy/<commit>/ios-deploy.zip`'
-          ' if you would like to codesign ios-deploy.zip, which has a google cloud bucket path of flutter_infra_release/ios-usb-dependencies/unsigned/ios-deploy/<commit>/ios-deploy.zip to be downloaded from \n',
+      kInputZipPathOption,
+      help:
+          'File path to the unsigned artifact zip file. The artifact should have been pre-fetched from gcs bucket through recipe \n',
     )
     ..addOption(
-      kGcsUploadPathOption,
-      help: 'The google cloud bucket path to upload the artifact to. \n'
-          'e.g. supply `--gcs-upload-path=gs://flutter_infra_release/ios-usb-dependencies/ios-deploy/<commit>/ios-deploy.zip`'
-          ' if you would like to codesign ios-deploy.zip, which has a google cloud bucket path of flutter_infra_release/ios-usb-dependencies/ios-deploy/<commit>/ios-deploy.zip to be uploaded to',
+      kOutputZipPathOption,
+      help: 'File path to store codesigned artifact zip file.'
+          ' The recipe will take codesigned zip from this location and upload back to gcs bucket \n',
```

Comment:
```suggestion
          ' The recipe will take codesigned zip from this location and upload back to gcs bucket.',
```

Reply:
Indeed, tested and it looks like the ending newline had no effect. Thanks for the suggestion Christopher!